server/express/index.js

```diff
@@ -28,7 +28,7 @@ app.post("/login", passport.authenticate(LOGIN_STRATEGY), (req, res) => {
 });
 
 app.get("/roles", passport.authenticate("jwt"), (req, res) => {
-  res.json(req.user.verifiedRoles);
+  res.json(req.user.useRoles);
```

Comment:
@nichonien could you explain this change when you have a moment? Did `passport-did-auth` change the name of the property on `user` from `verifiedRoles` to `useRoles`?

Reply:
@jrhender Yes, this is more aligned since it now has both valid and invalid roles. Here is our conversation - https://energywebfoundation.slack.com/archives/C028ZC4G2RE/p1665998261147449?thread_ts=1665991211.089449&cid=C028ZC4G2RE